README.md

```diff
@@ -36,11 +36,12 @@ The table below describes all the possible configuration variables, as well as t
 | `repositories:{full_name}:gitHubToken` | `string` | Token used to verify __outgoing__ requests to GitHub repository | ✓ | 
 | `repositories:{full_name}:thirdPartyFolders` | `string` | Comma-separated list of folders in which to look for changed files in pull request to remind user to update License. | X | `[]`
 | `repositories:{full_name}:claUrl` | `string` | The GitHub API URL to the CLA file in JSON form. See [here](https://developer.github.com/v3/repos/contents/#get-contents) for what the URL should look like. _Example:_ https://api.github.com/repos/AnalyticalGraphicsInc/cesium-concierge/contents/specs/data/config/CLA.json | X | _Disabled if not set._
-| `repositories:{full_name}:contributorsUrl` | `string` |  The GitHub API URL to `CONTRIBUTORS.md`. | X | _Disabled if not set._
+| `repositories:{full_name}:contributorsPath` | `string` |  Relative path to the `CONTRIBUTORS.md` file. | X | _Disabled if not set._
 | `repositories:{full_name}:maxDaysSinceUpdate` | `number` | "Bump" pull requests older than this number of days ago. | X | `30`
 | `repositories:{full_name}:unitTestPath` | `string` |  Relative path to the directory containing unit tests. _Example:`Specs/`_ | X | _Disabled if not set._
-| `port` | `number` | Port on which to listen to incoming requests | X | `5000`
-| `listenPath` | `string` | Path on which to listen for incoming requests | X | `"/"`
+| `port` | `number` | Port on which to listen to incoming requests. | X | `5000`
+| `listenPath` | `string` | Path on which to listen for incoming requests. | X | `"/"`
+| `outreachUsers` | `string` | Username(s) to mention when someone makes their first contribution. Example:`@OmarShehata`. You can add multiple people or prepend it with a message this way. | X | _Disabled if not set._
```

Comment:
` prepend it with a message this way` - a little vague, an example of this use case would help.

Reply:
Actually, do we need this value in the config file at all? Why not just specify it in the first time contribution message templates?